resources/constraints.yaml

```diff
@@ -5,6 +5,22 @@ test:
   max_runtime_seconds: 600
   cores: 4
 
+
+1m1c:
+  folds: 10
+  max_runtime_seconds: 60
+  cores: 1
+
+10m1c:
+  folds: 10
+  max_runtime_seconds: 600
+  cores: 1
+  
+1h1c:
+  folds: 10
+  max_runtime_seconds: 3600
+  cores: 1
+
```

Comment:
irrelevant for this PR.
Please only modify what is related for your framework.

Reply:
Sure. Thanks for the reminder! I am adding this because FLAML is optimizing for small resources. But I agree with what you suggested: can do this in a separate PR.